### PR TITLE
Add what REST API cannot do to README

### PR DESCRIPTION
**Pull request by dpriskorn**
Update readme with details about what the REST API cannot do

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +220,17 @@ These are features that Dolibarr does **not** yet fully support:
 - Payroll module
 - No native embedded Webmail, but you can send email to contacts in Dolibarr with e.g. offers, invoices, etc.
 - Dolibarr can't do coffee (yet)
+- The REST API currently cannot:
+  - Update a supplier order 
+  - Get tickets
```
Comment (eldy) [inline README.md:225]: Seems available in develop branch

File: README.md [hunk 2]
```diff
@@ -219,6 +220,17 @@ These are features that Dolibarr does **not** yet fully support:
...
+  - Add new ledger entry
+  - Output extrafields when getting orders
+  - Get virtual products (aka lots)
+  - Add special taxcodes to a line on a supplier invoice
+  - Get or post shipments
+  - Do updating or inserting of extrafields on any objects
```
Comment (eldy) [inline README.md:231]: Seems available in develop branch

File: README.md [hunk 3]
```diff
@@ -219,6 +220,17 @@ These are features that Dolibarr does **not** yet fully support:
 - Payroll module
 - No native embedded Webmail, but you can send email to contacts in Dolibarr with e.g. offers, invoices, etc.
 - Dolibarr can't do coffee (yet)
+- The REST API currently cannot:
+  - Update a supplier order 
+  - Get tickets
+  - Add new ledger entry
+  - Output extrafields when getting orders
```
Comment (eldy) [inline README.md:227]: Seems available in develop branch

File: README.md [hunk 4]
```diff
@@ -219,6 +220,17 @@ These are features that Dolibarr does **not** yet fully support:
...
+  - Get virtual products (aka lots)
+  - Add special taxcodes to a line on a supplier invoice
+  - Get or post shipments
+  - Do updating or inserting of extrafields on any objects
+  - Add an image to a product
```
Comment (eldy) [inline README.md:232]: Seems available in develop branch

File: README.md [hunk 5]
```diff
@@ -219,6 +220,17 @@ These are features that Dolibarr does **not** yet fully support:
...
+  - Add multiprices to products
```
Comment (eldy) [inline README.md:233]: Seems available in develop branch

**Comment by dpriskorn** [comment]
Here is another one :)
#18071
Maybe this can be done by updating the order?
https://github.com/Dolibarr/dolibarr/blob/1a63a379cc447c3ddc07dd1c9c715db9efe11141/htdocs/fourn/class/api_supplier_orders.class.php#L255
I would prefer a separate method and endpoint like "validate" though.

**Comment by dpriskorn** [comment]
Here is another one:
 #17058
ksar posted a solution in a comment

**Comment by dpriskorn** [comment]
Here is another one:
If prices are already present it should be optional to specify them IMO:
 #14764
this is labeled as a feature request, but IMO it is probably a bug because the user (me) expect Dolibarr to use the data it already has instead of defaulting to NULL.

File: README.md (hunk 4 above)
Comment (dpriskorn) [reply README.md:232]: Where? I found nothing in https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/product/class/api_products.class.php

File: README.md (hunk 5 above)
Comment (dpriskorn) [reply README.md:233]: Where? I found nothing in https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/product/class/api_products.class.php

File: README.md (hunk 3 above)
Comment (dpriskorn) [reply README.md:227]: Correct! Confirmed working on v13 also on get /orders/{id}

File: README.md (hunk 2 above)
Comment (dpriskorn) [reply README.md:231]: This line is too vague so I'll delete it

File: README.md (hunk 1 above)
Comment (dpriskorn) [reply README.md:225]: Correct! https://github.com/Dolibarr/dolibarr/blob/develop/htdocs/ticket/class/api_tickets.class.php
I'll remove it :)

**Comment by dpriskorn** [comment]
@eldy is it ready to merge now?

File: README.md (hunk 4 above)
Comment (eldy) [reply README.md]: A product image is sending a document image. You must use api document

**Pull request merged**